src/components/implementation/torrent/async_conn/as_connector.c

```diff
@@ -370,8 +370,9 @@ treadp(spdid_t spdid, td_t td, int *off, int *sz)
 	return ret;
 }
 
+// What are these definitions?
 int 
-twritep(spdid_t spdid, td_t td, int cbid, int sz)
+twritep(spdid_t spdid, td_t td, int cbid, int size, unsigned int sz)
```

Comment:
i don't know what the new variable is used for, the similar naming of 'size' and 'sz' is confusing.